### ISSUE TITLE
How to launch scrcpy without console?(run scrcpy-noconsole .vbs no respond.)

### ISSUE THREAD

**Issue by Liquidlkw** [Genymobile/scrcpy#2799]
- [y] I have read the [FAQ](https://github.com/Genymobile/scrcpy/blob/master/FAQ.md).
 - [y] I have searched in existing [issues](https://github.com/Genymobile/scrcpy/issues).

**Environment**
 - OS:   Windows
 - scrcpy version: 1.20
 - installation method: Windows release
 - device model:MEIZU 16T
 - Android version: 10

**Describe the bug**
I want to know how should I do to run the scrcpy-noconsole.vbs?
I hope that I can run the .vbs to launch scrcpy without console.However,It happen nothing when I run the .vbs.In othe side,I can run other .vbs by thre

On errors, please provide the output of the console (and `adb logcat` if relevant).

```
Please paste terminal output in a code block.
```

Please do not post screenshots of your terminal, just post the content as text instead.

**Comment by Liquidlkw** [comment]
@rom1v

**Comment by rom1v** [comment]
If nothing happens when you open without a console, it's probably because you get an error that you can't see because there is no console.

Run `scrcoy-console.bat` once to see any error.

**Comment by Liquidlkw** [comment]
![image](https://user-images.githubusercontent.com/48376042/142177793-d36b8bef-894a-4f73-99af-4c7674b2bded.png)
nothing happens when I run scrcoy-console.bat after run scrcpy-noconsole.vbs

**Comment by rom1v** [comment]
Could you translate what is written in the console for me, please?

_(also, please do not post screenshots of text, just the text)_

**Comment by rom1v** [comment]
> nothing happens when I run scrcoy-console.bat after run scrcpy-noconsole.vbs

Oh, so it's when you start several instances of scrcpy successively? If yes, please test #2783.

**Comment by npes87184** [comment]
> Could you translate in English what is written in the console for me, please?
> 
> _(also, please do not post screenshots of text, just the text)_

scrcpy.exe was not found or was not executable

**Comment by rom1v** [comment]
Is your Windows 64 bits?

Could you run `scrcpy -Vdebug` from a `cmd.exe` terminal?
https://github.com/Genymobile/scrcpy/blob/master/FAQ.md#command-line-on-windows

**Comment by Liquidlkw** [comment]
> > scrcpy.exe was not found or was not executable
> 
> Thank you for the translation 👍
> 
> @Liquidlkw
> 
> Is your Windows 64 bits?
> 
> Could you run `scrcpy -Vdebug` from a `cmd.exe` terminal? https://github.com/Genymobile/scrcpy/blob/master/FAQ.md#command-line-on-windows
yes,my Windows is 64 bits and I can run scrcpy -Vdebug from a cmd.exe terminal successfuliy

**Comment by Liquidlkw** [comment]
> > nothing happens when I run scrcoy-console.bat after run scrcpy-noconsole.vbs
> 
> Oh, so it's when you start several instances of scrcpy successively? If yes, please test #2783.

In the screenshot I  start several instances of scrcpy successively.However, I want to open only one instance without console now.So,should I continue try test #2783?

**Comment by Liquidlkw** [comment]
> > scrcpy.exe was not found or was not executable
> 
> Thank you for the translation 👍
> 
> @Liquidlkw
> 
> Is your Windows 64 bits?
> 
> Could you run `scrcpy -Vdebug` from a `cmd.exe` terminal? https://github.com/Genymobile/scrcpy/blob/master/FAQ.md#command-line-on-windows

yes,my Windows is 64 bits and I can run scrcpy -Vdebug from a cmd.exe terminal successfuliy

**Comment by Liquidlkw** [comment]
> > Could you translate in English what is written in the console for me, please?
> > _(also, please do not post screenshots of text, just the text)_
> 
> scrcpy.exe was not found or was not executable

Hello guy,thank you for your translation.
Are you Chinese ?
You can't open run scrcpy-noconsole .vbs ,too?

**Comment by Liquidlkw** [comment]
> > nothing happens when I run scrcoy-console.bat after run scrcpy-noconsole.vbs
> 
> Oh, so it's when you start several instances of scrcpy successively? If yes, please test #2783.

I try to replace two binarys ,but I also run scrcpy-noconsole .vbs.
